spec/unit/shoulda/matchers/active_record/define_enum_for_matcher_spec.rb

```diff
@@ -33,19 +59,30 @@ def self.statuses; end
       end
 
       it "rejects a record where the attribute is not defined as an enum" do
-        message = "Expected #{record_with_array_values.class} to define :#{non_enum_attribute} as an enum"
+        message = "Expected #{record_with_array_values.class} to define :#{non_enum_attribute} as an enum and store the value in a column with an integer type"
 
-        expect do
-          expect(record_with_array_values).to define_enum_for(non_enum_attribute)
-        end.to fail_with_message(message)
+        expect {
```

Comment:
I've been slowly changing the existing codebase to use this style:

``` ruby
message = ...

assertion = lambda do
  expect(record_with_array_values).
    to define_enum_for(non_enum_attribute)
end

expect(&assertion).to fail_with_message(message)
```

What do you think about changing the tests you've modified to use this style?